src/gwt/src/org/rstudio/studio/client/workbench/views/vcs/git/GitStatusRenderer.java

```diff
@@ -98,6 +98,7 @@ private String descForStatus(char c)
    {
       switch (c)
       {
+         // TODO Unsure about these 'case's, might need to leave be for the time
```

Comment:
You can localize the return strings here. Don't localize the case values. These strings show up in the Git UI if you hover the mouse over the status graphic (e.g. [M] in screenshot below).

Lower in this file, the `imgForStatus()` function is returning those graphics which contain a letter, such as the [M]. We aren't localizing those graphics, but having the tooltip localized would be good.

![Screen Shot 2021-12-08 at 16 52 06](https://user-images.githubusercontent.com/10569626/145314852-cdc4872d-6036-4623-affd-b7cd75533dfb.png)